package.json

```diff
@@ -21,7 +21,7 @@
     "test": "cross-env BABEL_ENV=test NODE_PATH=./src nyc -s mocha -R dot --recursive tests/src -t 5s",
     "test-compiled": "npm run prepublish && NODE_PATH=./lib mocha --compilers js:babel-register --recursive tests/src",
     "test-all": "npm test && for resolver in ./resolvers/*; do cd $resolver && npm test && cd ../..; done",
-    "prepublish": "gulp prepublish",
+    "prepare": "gulp prepublish",
```

Comment:
please revert this; "prepublish" is the correct script name, despite npm's badly worded warning.

Reply:
https://www.balena.io/blog/safely-migrating-away-from-prepublish-with-npm-4/

It's deprecated. I don't understand "badly worded warning". NPM's developers clearly described why this script is deprecated and how one can update it to new script name.

Reply:
There's no value in doing so; "prepublish" works in every version of npm. I doubt npm will ever remove it, and if/when they do, we can handle it then.

Reply:
```suggestion
    "prepublish": "gulp prepublish",
```